src/components/ThemeProvider/__stories__/ThemeProvider.stories.helpers.tsx

```diff
@@ -201,3 +201,48 @@ export const UsageGuidelinesThemeProvider = () => (
     ]}
   />
 );
+
+export const DescriptionWithLinkMondaySdkIntegration = () => (
+  <>
+    In case you are developing an external application for monday.com. You can use ThemeProvider in combination with the{" "}
+    <Link
+      href="https://developer.monday.com/apps/docs/mondayget#sample-context-objects-for-each-feature-type"
+      withoutSpacing
+    >
+      monday.com SDK
+    </Link>
+    , to apply monday.com <b>system</b> and <b>product</b> themes to your application. This will allow your application
+    to be consistent with the monday.com UI.
+  </>
+);
```

Comment:
```suggestion
export const DescriptionWithLinkMondaySdkIntegration = () => (
  <>
    When developing an external application for monday.com (iframe). You can use <code><ThemeProvider/></code> in combination with the{" "}
    <Link
      href="https://developer.monday.com/apps/docs/mondayget#sample-context-objects-for-each-feature-type"
      withoutSpacing
    >
      monday.com SDK
    </Link>
    , to apply monday.com <b>system</b> and <b>product</b> themes to your application. This will allow your application
    to be consistent with the monday.com UI.
  </>
);
```